apps/champions/priv/skills.json

```diff
@@ -20,35 +1663,339 @@
             "energy_recharge": 50,
             "delay": 0
           }
         ],
         "target_strategy": "random",
+        "target_count": 1,
+        "target_allies": false
+      }
+    ],
+    "cooldown": 5
+  },
+  {
+    "name": "Valtimer Ultimate",
+    "energy_regen": 0,
+    "animation_duration": 0,
+    "animation_trigger": 0,
+    "effects": [
+      {
+        "type": {
+          "period": 1
+        },
+        "initial_delay": 0,
+        "components": [],
+        "modifiers": [],
+        "executions": [
+          {
+            "type": "DealDamage",
+            "attack_ratio": 3.26,
+            "energy_recharge": 50,
+            "delay": 0
+          },
+          {
+            "type": "AddEnergy",
+            "amount": -200
+          }
+        ],
+        "target_strategy": {
+          "highest": "Attack"
+        },
+        "target_count": 1,
+        "target_allies": false
+      }
+    ]
+  },
+  {
+    "name": "H4ck Basic",
+    "energy_regen": 100,
+    "animation_duration": 0,
+    "animation_trigger": 0,
+    "effects": [
+      {
+        "type": {
+          "type": "instant"
+        },
+        "initial_delay": 0,
+        "components": [],
+        "modifier": [],
+        "executions": [
+          {
+            "type": "DealDamage",
+            "attack_ratio": 1,
+            "energy_recharge": 50,
+            "delay": 0
+          }
+        ],
+        "target_strategy": "backline",
+        "target_count": 1,
+        "target_allies": false
+      }
+    ],
+    "cooldown": 5
+  },
+  {
+    "name": "H4ck Ultimate",
+    "energy_regen": 0,
+    "animation_duration": 0,
+    "animation_trigger": 0,
+    "effects": [
+      {
+        "type": {
+          "duration": 2,
+          "period": 1
+        },
+        "initial_delay": 0,
+        "components": [],
+        "modifiers": [
+          {
+            "attribute": "speed",
+            "modifier_operation": "Multiply",
+            "magnitude_calc_type": "Float",
+            "float_magnitude": 0
+          }
+        ],
+        "executions": [
+          {
+            "type": "DealDamage",
+            "attack_ratio": 0.49,
+            "energy_recharge": 50,
+            "delay": 0
+          }
+        ],
+        "target_strategy": "all",
+        "target_count": 6,
+        "target_allies": false
+      }
+    ]
+  },
+  {
+    "name": "Kenzu Basic",
+    "energy_regen": 100,
+    "animation_duration": 0,
+    "animation_trigger": 0,
+    "effects": [
+      {
+        "type": {
+          "type": "instant"
+        },
+        "initial_delay": 0,
+        "components": [],
+        "modifier": [],
+        "executions": [
+          {
+            "type": "DealDamage",
+            "attack_ratio": 0.8,
+            "energy_recharge": 50,
+            "delay": 0
+          },
+          {
+            "type": "DealDamage",
+            "attack_ratio": 0.8,
+            "energy_recharge": 50,
+            "delay": 0
```

Comment:
We can ask for the correct value later, but we should set these delays to increasing numbers like 0, 1 to correctly emulate the spirit of the skill

Reply:
Yeah, that's something to be defined yet, I left them in 0 so they don't interfere with what we're currently testing

Reply:
What I mean is that we can leave some default "1, 2, 3" values, we can change fix them again later

Reply:
They won't interfere with anything because delay is ignored for DealDamage executions